packages/gatsby-theme-docs/src/components/beta-flag.js

```diff
@@ -41,7 +41,7 @@ const betaHint =
 const BetaFlag = (props) => {
   if (props.href) {
     return (
-      <Link href={props.href} noUnderline={true} css={getStyles(props)}>
+      <Link href={props.href} nounderline={true} css={getStyles(props)}>
```

Comment:
React runtime warning:   custom HTML attributes must be lowercase (ideally prefixed "data-" but that would have made other code messy)